packages/utils/src/conversions.ts

```diff
@@ -148,41 +160,45 @@ export function leBigIntToBuffer(n: bigint, size?: number): Buffer {
 
 /**
  * Converts a bigint to a buffer. Alias for beBigIntToBuffer.
- * @param n The bigint to convert.
+ * @param value The bigint to convert.
  * @returns The buffer representation of the bigint.
  */
-export function bigIntToBuffer(n: bigint): Buffer {
-    return beBigIntToBuffer(n)
+export function bigIntToBuffer(value: bigint): Buffer {
+    return beBigIntToBuffer(value)
 }
 
 /**
  * Converts a BigNumberish type to a bigint. If the input is already a bigint,
  * the return value will be the bigint itself, otherwise it will be converted
  * to a bigint using big-endian byte order.
- * @param n The BigNumberish value to convert.
+ * @param value The BigNumberish value to convert.
  * @returns The bigint representation of the BigNumberish value.
  */
-export function bigNumberishToBigInt(n: BigNumberish): bigint {
-    if (isNumber(n) || isBigInt(n) || isStringifiedBigInt(n) || isHexadecimal(n)) {
-        return BigInt(n as BigNumber | number)
+export function bigNumberishToBigInt(value: BigNumberish): bigint {
```

Comment:
If you're going to accept Uint8Arrays elsewhere, it becomes a bit inconsistent that the Bignumberish type doesn't include them.

Reply:
You're right! I'll add it there as well